internal/controller/storageclient_controller.go

```diff
@@ -697,6 +702,43 @@ func removeStorageClaimAsOwner(obj client.Object) {
 	}
 }
 
+func (r *storageClientReconcile) reconcileResourcesByGVK(
+	resources [][]byte,
```

Comment:
I think we decided to define a type name for this complex type

Reply:
we only have a type alias for a map and this part is only the value of the map which isn't aliased further.

Reply:
preferably have an alias for the type []Bytes, this will allow us to have meaningful names for both the map and the slices